Formula/postgresql@9.6.rb

```diff
@@ -0,0 +1,150 @@
+class PostgresqlAT96 < Formula
+  desc "Object-relational database system"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v9.6.5/postgresql-9.6.5.tar.bz2"
+  sha256 "06da12a7e3dddeb803962af8309fa06da9d6989f49e22865335f0a14bad0744c"
+
+  bottle do
```

Comment:
It's a new formula so there shouldn't be a bottle block.

Reply:
I've removed the bottles.

Reply:
I still see them …

Reply:
Argh, I removed them on another machine and then rebased against origin/master without pulling in my changes from my forks' remote first, so I added them back.

Removed (again).